pkgs/development/python-modules/py-slvs/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  swig,
+  cmake,
+  ninja,
+  setuptools,
+  scikit-build,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "py-slvs";
+  version = "1.0.6";
+
+  src = python3Packages.fetchPypi {
+    pname = "py_slvs";
+    inherit version;
+    sha256 = "sha256-U6T/aXy0JTC1ptL5oBmch0ytSPmIkRA8XOi31NpArnI=";
+  };
+
+  pyproject = true;
+
+  nativeBuildInputs = with pkgs; [
+    swig
+  ];
+
+  build-system = [
+    cmake
+    ninja
+    setuptools
+    scikit-build
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  meta = with lib; {
+    description = "Python binding of SOLVESPACE geometry constraint solver";
+    homepage = "https://github.com/realthunder/slvs_py";
+    license = licenses.gpl3;
```

```suggestion
    license = lib.licenses.gpl3;
```

also please use gpl3Only or gpl3Plus in accordance with upstream